averbis/core/_rest_client.py

```diff
@@ -880,9 +883,9 @@ def get_process_state(self) -> ProcessState:
 
     def export_text_analysis(
         self,
-        annotation_types: str = None,
-        page: Union[int, None] = None,
-        page_size: Union[int, None] = 100,
+        annotation_types: Optional[str] = None,
+        page: Optional[int] = None,
+        page_size: Optional[int] = 100,
```

Comment:
I don't think the Optional is necessary here, but we can directly set the type to **int** (cf https://docs.python.org/3/library/typing.html#typing.Optional).